### DISCUSSION TITLE
Welcome to indie-dev-toolkit Discussions!

### DISCUSSION THREAD

**Discussion by thedaviddias** [thedaviddias/indie-dev-toolkit#1] [Announcements]
## 👋 Welcome Indie Hackers!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about and about the tools shared in this project.
  * Share any ideas or experience you had with those tools.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AlexanderKvamme** [comment]
Hey guys. Looks like im starting this thread:P I'm a developer from Norway that loves building products. I built a feature requests manager specifically for Indie Hackers and would love to be considered for the list! It's on par with WishKit but a lot cheaper so that developers trying different ideas can gather information from their users! Check it out at https://featurefest.dev!
